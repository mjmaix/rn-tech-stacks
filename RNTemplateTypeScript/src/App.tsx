import React from "react";

import AppNav from "./AppNav";

export default class App extends React.Component<{}> {
  public render() {
    return <AppNav />;
  }
}
