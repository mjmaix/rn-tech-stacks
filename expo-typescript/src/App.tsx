import * as React from 'react';
import { Text, View } from 'react-native';

export default () => (
  <View style={{ margin: 50 }}>
    <Text>Hello Expo!</Text>
  </View>
);
