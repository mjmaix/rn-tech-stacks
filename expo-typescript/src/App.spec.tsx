import * as React from 'react';
import * as renderer from 'react-test-renderer';

import App from './App';

describe('App', () => {
  it('snapshot test', () => {
    // test auto prettier
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
