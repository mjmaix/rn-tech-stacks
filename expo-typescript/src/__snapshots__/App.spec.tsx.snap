// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<View
  style={
    Object {
      "margin": 50,
    }
  }
>
  <Text>
    Hello Expo!
  </Text>
</View>
`;
